Package.swift

```diff
@@ -1,33 +1,33 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.6
```

Comment:
So I wasn't going to update Xcode until Monterey was released. Was there a code change that required the update or you were just using Xcode 14 locally?

Reply:
It was a combination of concurrency support and fixing the `@main` bug. Xcode 13.3/13.4 would work too.

Reply:
Can we make it 13.4.1? It's what I build Xcode 1.0 with

Reply:
Done 👍🏼

Reply:
> So I wasn't going to update Xcode until Monterey was released.

Did you mean Ventura? Because I already bumped macOS from Big Sur to Montrey

Reply:
I do mean Ventura.. I need more coffee.